docs/getting-started/src/main/java/sample/ElectronicMoney.java

```diff
@@ -1,19 +1,25 @@
 package sample;
 
+import com.scalar.db.config.DatabaseConfig;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
 import java.util.Properties;
 
 public abstract class ElectronicMoney {
+  private static final String SCALARDB_PROPERTIES = "scalardb.properties";
   protected static final String NAMESPACE = "emoney";
   protected static final String TABLENAME = "account";
   protected static final String ID = "id";
   protected static final String BALANCE = "balance";
-  protected Properties props;
+  protected DatabaseConfig dbConfig;
 
-  public ElectronicMoney() {
-    props = new Properties();
-    props.setProperty("scalar.db.contact_points", "localhost");
-    props.setProperty("scalar.db.username", "cassandra");
-    props.setProperty("scalar.db.password", "cassandra");
+  public ElectronicMoney() throws IOException {
+    dbConfig =
+            new DatabaseConfig(
+                    new File(getClass().getClassLoader().getResource(SCALARDB_PROPERTIES).getFile()));
```

Comment:
> Why reading from file?

For easily executing sample applications with Cassandra and cosmos db.
As per the previous implementation, the Cassandra configuration is hardcoded here.

Reply:
@feeblefakie 
Sorry for the inconvenience.
Property file moved to `getting-started` directory.
Now only the property file available here because configuration details exist in the document.